styles/all/template/webpush.js

```diff
@@ -0,0 +1,327 @@
+/* global phpbb */
+
+'use strict';
+
+/**
+ * Convert raw key ArrayBuffer to base64 string.
+ *
+ * @param {ArrayBuffer} rawKey Raw key array buffer as exported by SubtleCrypto exportKey()
+ * @returns {string} Base64 encoded raw key string
+ */
+phpbb.rawKeyToBase64 = (rawKey) => {
```

Comment:
Do these base64 functions need to be in the ACP JS file instead? It looks like they're only used in that file, and I'm not sure they will be accessible in the ACP if they are in this file.